R/pkg/R/DataFrame.R

```diff
@@ -2200,4 +2200,101 @@ setMethod("coltypes",
             rTypes[naIndices] <- types[naIndices]
 
             rTypes
+          })
+
+#' Display the structure of a DataFrame, including column names, column types, as well as a
+#' a small sample of rows.
+#' @name str
+#' @title Compactly display the structure of a dataset
+#' @rdname str_data_frame
+#' @family dataframe_funcs
```

Comment:
This has been updated recently - you should see when you rebase the latest in master - it would be `#' @family DataFrame functions`